### DISCUSSION TITLE
quic ipv6

### DISCUSSION THREAD

**Discussion by ValorBao** [Tencent/tquic#120] [Q&A (问题和回答)] [answered]
可以提供quic ipv6的例子吗

**Comment by wangfuyu** [comment]
> 可以提供quic ipv6的例子吗

TQUIC仅作为QUIC协议栈，不会限制IP版本，socket是由集成的框架提供的，比如将TQUIC集成到Nginx框架中，是否支持IPv6监听是Nginx决定的哈。

**Comment by iyangsj** [comment, marked as answer]
见 https://github.com/Tencent/tquic/tree/develop/tools/src/bin

使用方式具体可参考 : https://tquic.net/zh/docs/getting_started/demo ,  地址换成IPv6地址即可，例如 [::1]

```
./tquic_server -c cert.crt -k cert.key -l [::1]:8443

./tquic_client -c [::1]:8443 https://exmaple.org 
```
